server/form-pages/assess/matchingInformation/matchingInformationTask/matchingInformation.ts

```diff
@@ -165,7 +167,10 @@ export default class MatchingInformation implements TasklistPage {
   }
 
   get suggestedLengthOfStay() {
-    return placementDurationFromApplication(this.assessment.application)
+    return formatDuration(daysToWeeksAndDays(placementDurationFromApplication(this.assessment.application)), {
```

Comment:
🤯 what a handy function. 
I think it would be good to have this as a static method on DateFormats so we don't have to duplicate the options each time

Reply:
Good call. I've added a refactor in the last commit